src/components/interfaces/MOBILE_API.xml

```diff
@@ -1314,6 +1314,8 @@
         <description>Reflects the current primary audio source (if selected).</description>
         <element name="NO_SOURCE_SELECTED">
         </element>
+        <element name="CD">
+        </element>
```

Comment:
Add `since="5.0"` to new enum values

Reply:
@jacobkeeler Fixed at commit fd9f349304e4d47d25d1fed79a0442a599717de8